### ISSUE TITLE
Creative Commons License Not Recommended For Software

### ISSUE THREAD

**Issue by buzzbyte** [helblazer811/CyclistSocialNetwork#2]
This software is licensed under a license not meant for software, as the Creative Commons' licenses "do not contain specific terms about the distribution of source code" and are also not compatible with other major software licenses (See: https://wiki.creativecommons.org/index.php/Frequently_Asked_Questions#Can_I_apply_a_Creative_Commons_license_to_software.3F ).

I would recommend assigning a free software license such as the GNU General Public License, MIT License, or any of the BSD licenses.

**Comment by helblazer811** [comment]
Yeah. I know about the common software licences, but they don't really apply to websites as much as other things because otherwise you would have ten copies of the same website on the internet at the same time.
